DetectorDescription/DDCMS/plugins/dd4hep/DDDefinitions2Objects.cc

```diff
@@ -2243,9 +2243,6 @@ static long load_dddefinition(Detector& det, xml_h element) {
       if (mfv1.isValid())
         wv.placeVolume(mfv1, 1);
 
-      // Can not deal with reflections without closed geometry
-      det.manager().CloseGeometry();
```

Comment:
@ghugo83 - there is no harm in closing geometry earlier. DD4hep will close it any way. At least we'd know when we forbid any further modifications to geometry.

Reply:
@ianna It looks like we need to close the geometry.
When I revert this line, the tests pass.